vexim/adminaliaschange.php

```diff
@@ -99,6 +108,9 @@
               } ?> class="textfield">
             </td>
           </tr>
+          <?php
+           if ($domrow['avscan'] == "1") {
+        ?>
```

Comment:
What do you mean, to disable the field but keeping it visible? Or to enforce a default spam/antivirus-check?